SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -82,6 +82,8 @@ SCDocHTMLRenderer {
 			};
 		};
 
+		if(thisProcess.platform.name == \windows) {result = result.replace("\\", "/")}; // replace backward with forward slash on Windows
+
```

Comment:
Couldn't we make this somehow more universal via

```
// enforce unix path for html
result = result.replace(Platform.pathSeparator, "/");
```

Platform dependent if statements can lead to some trouble while testing, e.g. invoking methods which have been deleted, but since I would not be on windows the testing suite wouldn't complain.

Reply:
```
result = result.replace(Platform.pathSeparator, "/");
```
This looks cleaner indeed, but is redundant on Unix. Do you think it's fine to always run it?

> invoking methods which have been deleted

I'm not sure I understand?

Reply:
@capital-G I applied the suggest change.